src/bioregistry/data/bioregistry.json

```
@@ -96,7 +96,11 @@
     "synonyms": [
       "4DN"
     ],
-    "uri_format": "https://data.4dnucleome.org/biosources/$1"
+    "uri_format": ["https://data.4dnucleome.org/biosources/$1"],
```

```suggestion
    "uri_format": "https://data.4dnucleome.org/biosources/$1",
````

seems like the issue was just a missing comma